docs/components/integration-framework/introduction.md

```diff
@@ -9,44 +9,51 @@ Camunda offers an **integration framework** which allows you to integrate extern
+A [job worker](../concepts/job-workers.md) is a service capable of performing a particular task in a process.
 
-You can implement a **job worker**, which performs the integration with an external system. For example, the **job worker** code could implement sending a message to an external system. When started, the **job worker** would [request jobs of a certain type by polling](../concepts/job-workers.md#long-polling) a specified Camunda Platform 8 instance. The processes on this Camunda Platform 8 instance would create jobs by specifying the respective job type in BPMN tasks (for example, in a [service task via task definition](../../modeler/bpmn/service-tasks#task-definition)).
+You can implement a job worker, which performs the integration with an external system. For example, the job worker code could implement sending a message to an external system. When started, the job worker would [request jobs of a certain type by polling](../concepts/job-workers.md#long-polling) a specified Camunda Platform 8 instance. The processes on this Camunda Platform 8 instance would create jobs by specifying the respective job type in BPMN tasks (for example, in a [service task via task definition](../../modeler/bpmn/service-tasks#task-definition)).
 
-The integration with external systems using **job workers** requires you to:
+The integration with external systems using job workers requires you to:
 
-1. Implement and run the respective **job worker**.
+1. Implement and run the respective job worker.
 2. Specify the respective job types in the BPMN process.
 
-#### Next steps to get started with job workers
+### Next steps to get started with job workers
 
 - [Learn about job workers](../concepts/job-workers.md)
 - [Implement a job worker which subscribes to a certain task](../../best-practices/development/connecting-the-workflow-engine-with-your-world#subscribing-to-tasks-using-a-job-worker)
 
-### Connectors
+## Connectors
 
 A **Connector** is a reusable building block that performs the integration with an external system and works out of the box. It is represented as a concrete task in a [BPMN process](../concepts/processes.md), which can be configured with domain specific parameters without implementing custom business logic.
 
-The concept of a **Connector** consists of two parts:
+The concept of a Connector consists of two parts:
 
-1. The business logic is implemented following the [job worker](/components/concepts/job-workers.md) pattern.
+1. The business logic is implemented by a [Connector function](./connectors/custom-built-connectors/connector-sdk.md#runtime-logic).
 2. The user interface during modeling is provided using a [Connector template](./connectors/custom-built-connectors/connector-templates.md).
 
-You can use **Connectors** from two different perspectives:
+You can use different sets of Connectors, depending on the environment you are using:
 
-#### Out-of-the-box Connectors
+### Camunda Platform 8 SaaS
```

Comment:
I don't think we should change our headings. These headings are used in a number of other areas and should be kept.
We can describe the environment where these headings will work on the page.

Reply:
I actually checked beforehand and we at least don't use the two headings to link to them from within our documentation. I simply think the headings don't convey the right message here, that's why I think we should change them. Just because we may have external resources that link to this shouldn't mean we never change them. If we are aware of any resources that link to this, let's adjust them. If not, they'll pop up eventually. Yes, it's not ideal, but that should not stop us from improving the docs.

Reply:
We can surely change the headers to be more meaningful and keep old (local) links functioning.